modules/serverless-install-kafka-odc.adoc

```diff
@@ -18,9 +18,8 @@ Cluster administrators can enable the use of Apache Kafka functionality in an {S
 
 . In the *Administrator* perspective, navigate to *Operators* → *Installed Operators*.
 . Check that the *Project* dropdown at the top of the page is set to *Project: knative-eventing*.
-. Click *Knative Kafka* in the list of *Provided APIs* for the {ServerlessOperatorName} to go to the *Knative Kafka* tab.
-. Click *Create Knative Kafka*.
-. Optional: Configure the *KnativeKafka* object in the *Create Knative Kafka* page. To do so, use either the default form provided or edit the YAML.
+. In the list of *Provided APIs* for the {ServerlessOperatorName}, find the *Knative Kafka* box and click *Create Instance*.
+. Configure the *KnativeKafka* object in the *Create Knative Kafka* page. To successfully install *KnativeKafka*, you need to at least specify the bootstrap servers and enable installation of KafkaChannel and KafkaSource. You can use either the default form provided or edit the YAML.
```

Comment:
```suggestion
. Configure the *KnativeKafka* object in the *Create Knative Kafka* page.
+
[IMPORTANT]
====
To use the Kafka channel or Kafka source on your cluster, you must toggle the *Enable* switch for the options you want to use to *true*. These switches are set to *false* by default.
====
```

Is this better @matzew ? I think it should also take care of https://issues.redhat.com/browse/SRVKE-708 if we just mention enabling the source and channel here and discuss the bootstrap part in the docs for the source or channel themselves.

The bootstrap part is covered for the source but not the channel at the minute in the docs:

- Source: https://srvls-kafka-setup-updates--ocpdocs.netlify.app/openshift-enterprise/latest/serverless/event_sources/serverless-kafka-source.html#serverless-kafka-source-odc_serverless-kafka-source
- Channel: https://srvls-kafka-setup-updates--ocpdocs.netlify.app/openshift-enterprise/latest/serverless/event_workflows/serverless-channels.html#serverless-create-kafka-channel-yaml_serverless-channels

So, if these are the wrong way round maybe this needs to be modified, but afaik this was the information that was given to me at the time when it was documented, and was SME and QE verified, so maybe you meant the `KafkaSource` in the comment above?

Reply:
for installing the channel, we have to set the bootstrap server:
https://docs.openshift.com/container-platform/4.6/serverless/serverless-kafka.html

That's a global setting (for the channel).  Each instance of a `KafkaChannel` does not need the info.

That's correct

Reply:
Thanks for the feedback Matthias. This goes in line with what we see in the UI.

Ashleigh, the current mention of bootstrap servers for source and lack of it for channel makes sense. When creating a source, it does ask for bootstrap servers. However, when creating the channels, we don't need to specify it, as the bootstrap servers specification is borrowed from KnativeKafka setup.

Reply:
And, I implemented your version with the [IMPORTANT] note.